tools/webpack/packages.js

```diff
@@ -43,149 +31,24 @@ const gutenbergPackages = Object.keys( dependencies )
-				preset: [
-					'default',
-					{
-						discardComments: {
-							removeAll: true,
-						},
-					},
-				],
-			} ),
-		] )
-			.process( content, {
-				from: 'src/app.css',
-				to: 'dest/app.css',
-			} )
-			.then( ( result ) => result.css );
-	}
-	return content;
-};
-
-/*
- * Matches a block's name in paths in the form
- * build-module/<blockName>/view.js
- */
-const blockNameRegex = new RegExp( /(?<=build-module\/).*(?=(\/view))/g );
-
-const createEntrypoints = () => {
-	/*
-	 * Returns an array of paths to view.js files within the `@wordpress/block-library` package.
-	 * These paths can be matched by the regex `blockNameRegex` in order to extract
-	 * the block's name.
-	 *
-	 * Returns an empty array if no files were found.
-	 */
-	const blockViewScriptPaths = fastGlob.sync(
-		'./packages/block-library/build-module/**/view.js'
-	);
-
-	/*
-	 * Go through the paths found above, in order to define webpack entry points for
-	 * each block's view.js file.
-	 */
-	const blockViewScriptEntries = blockViewScriptPaths.reduce(
-		( entries, scriptPath ) => {
-			const [ blockName ] = scriptPath.match( blockNameRegex );
-
-			return {
-				...entries,
-				[ 'blocks/' + blockName ]: scriptPath,
-			};
-		},
-		{}
-	);
-
-	const packageEntries = gutenbergPackages.reduce( ( memo, packageName ) => {
+module.exports = {
+	...baseConfig,
+	name: 'packages',
+	entry: gutenbergPackages.reduce( ( memo, packageName ) => {
 		return {
 			...memo,
 			[ packageName ]: `./packages/${ packageName }`,
 		};
-	}, {} );
-
-	return { ...packageEntries, ...blockViewScriptEntries };
-};
-
-module.exports = {
-	optimization: {
-		// Only concatenate modules in production, when not analyzing bundles.
-		concatenateModules:
-			mode === 'production' && ! process.env.WP_BUNDLE_ANALYZER,
-		minimizer: [
-			new TerserPlugin( {
-				cache: true,
-				parallel: true,
-				sourceMap: mode !== 'production',
-				terserOptions: {
-					output: {
-						comments: /translators:/i,
-					},
-					compress: {
-						passes: 2,
-					},
-					mangle: {
-						reserved: [ '__', '_n', '_nx', '_x' ],
-					},
-				},
-				extractComments: false,
-			} ),
-		],
-	},
-	mode,
-	entry: createEntrypoints(),
+	}, {} ),
 	output: {
 		devtoolNamespace: 'wp',
-		filename: ( pathData ) => {
-			const { chunk } = pathData;
-			const { entryModule } = chunk;
-			const { rawRequest, rootModule } = entryModule;
-
-			// When processing ESM files, the requested path
-			// is defined in `entryModule.rootModule.rawRequest`, instead of
-			// being present in `entryModule.rawRequest`.
-			// In the context of frontend view files, they would be processed
-			// as ESM if they use `import` or `export` within it.
-			const request = rootModule?.rawRequest || rawRequest;
-
-			if ( request.includes( '/view.js' ) ) {
-				return `./build/block-library/[name]/view.min.js`;
-			}
-
-			return `./build/[name]/index.min.js`;
-		},
+		filename: './build/[name]/index.min.js',
```

Comment:
Perfect, love to see this super polished. 🎉

Reply:
Yes, when we have two configs some parts become simpler 😄